api/api/src/review_handler.rs

```diff
@@ -20,85 +25,83 @@ pub fn list_reviews_handler(demo_mode: Result<DummyResponse, NetworkResponse>) -
     serde_json::to_string(&response).expect("Return 500 internal server error.")
 }
 
-/// Takes in a `review_id` and returns a 200 OK with the associated review as JSON
-/// otherwise, a 404 NotFound NetworkResponse.
 #[get("/<review_id>")]
 pub fn list_review_handler(
-    review_id: u32,
+    review_id: i32,
```

Comment:
Why is review an `i32` now? Is there a reason it should be signed?

Reply:
Diesel doesn't seem to want to put anything other than an i32 into the database.

Reply:
Looks like postgresql doesn't support unsigned integers? I guess this is fine